### ISSUE TITLE
NTRs: supramolecular structure/complex, protein fibril

### ISSUE THREAD

**Issue by paolaroncaglia** [geneontology/go-ontology#12099]
(Issue opened by Paola, rather heavily edited by @dosumis  in order to give broader context on supramolecular complex).
## NTR: supramolecular complex:

Suggested def: "A cellular component that consists of an indeterminate number of proteins or macromolecular complexes, organized into a regular, higher-order structure such as a polymer, sheet, network or a fiber."

 Examples: microtubule, actin filament, collagen sheet, collagen fibril, chromatin

Having this will allow us to separate out content that is out of scope for IntAct and so is not under the purview of planned refactoring under discussion in the protein complex working group. It is also better aligned with the terminology used by biologists - most biologists would not refer to the examples listed above as protein complexes. 'supramolecular complex' has precedent in the ECM lit.

Issues
1.  ribosome subunit is among macromolecular complex terms that are out of scope for IntAct, and should have a new classification.  But Harold (GH name?) points out that these have a determinate number of components in any one species (ref?).  So these will cannot be classified under supramolecular structure unless the draft def changes.
2.  Need to consider relationship to proposed PRO term 'protein aggregate' (worked on by @addiehl ).  One intended use of this term is as a parent class for 'immune complex'. This is not in GO, but arguably should be.
3.   Karen Christie (GH name = ?) commented: “I was wondering whether this is intended to cover things like keratohyalin granules (storage structures for the precursor proteins which are processed to form the stratum corneum), where there is a structure composed of an indeterminate number of things, but where I think you might not be able to say it has a higher order structure. It seems we need something that can cover these sorts of less organized large scale structures.”
## NTR: protein fibril

Stan requested this term via TG FF:

id: GO:1990870
name: protein fibril
namespace: cellular_component
def: "A polymer of proteins that form a fine fiber." [PMID:12764608]
is_a: GO:0043234 ! protein complex

This is a very general class under 'supramolecular complex' (given draft def) and which covers microtubules, actin filaments and probably collagen fibrils too. There doesn't seem to be a very strong case for making such a grouping, but, it works for Stan's use case (if that use case is in scope for GO - see below).  It would be good to have a way to define this more formally.  PATO already has 'fiber shaped'.  We can request polymeric from PATO too.

Re - Stan's use case: it is unclear if the structure observed in PMID:12764608 is pathological only and therefore out of scope; this may not always be the case. 

Stan’s note: “It seems that “protein fibrils” may be pathological in some cases and normal in others (zona pellucida, PMID:26043223).  It may be similar to “neurofibrillary tangle” (GO:0097418), which seems to be mostly pathological.  Maybe “protein aggregate” would be a good intermediate term to place between “macromolecular complex” (GO:0032991) and a potential protein fibril term.”

Note: Stan’s ‘protein fibril’ request on TGFF has been obsoleted.

References: 
- email thread 'protein fibril' on the ontology mailing list
- minutes of today’s editors call: 
  http://wiki.geneontology.org/index.php/Ontology_meeting_2015-10-15#Supramolecular_complex
  http://wiki.geneontology.org/index.php/Ontology_meeting_2015-10-15#Related_.2F_Follow-up:_protein_fibril

Thanks!

Paola

**Comment by paolaroncaglia** [comment]
I just obsoleted 'protein fibril'.

**Comment by bmeldal** [comment]
Collagens are definitely fibrils made up of multiples of defined trimers.

David and I worked with Sylvie from MatrixDB last year to tidy up the collagens into their morphological groups. 
GO:0005581 collagen trimer
Def: A protein complex consisting of three collagen chains assembled into a left-handed triple helix. These trimers typically assemble into higher order structures.

And its higher order structures:
GO:0098644 complex of collagen trimers
Def: A complex of collagen trimers such as a fibril or collagen network. 

...which actually is not a complex and needs to be reworked.
We discussed this briefly last week with David in the complex ontology WG.

On the other hand, things like amyloid fibers (fibriles?) are definitely pathological. 

So I think Stan & Karen were both right with their general observations but I haven't read Stan's paper to know the context of his term request.

Birgit
